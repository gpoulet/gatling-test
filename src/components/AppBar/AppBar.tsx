import React from "react";
import Navbar from "react-bootstrap/Navbar";
import Nav from "react-bootstrap/Nav";
import {ROUTE_HOME} from "../../constants";

const AppBar = () => {
    return (
        <Navbar bg="dark" variant="dark">
            <Navbar.Brand href="/">Gatling test</Navbar.Brand>
            <Nav className="mr-auto">
                <Nav.Link href={ROUTE_HOME}>Home</Nav.Link>
            </Nav>
        </Navbar>
    )
}

export default AppBar;