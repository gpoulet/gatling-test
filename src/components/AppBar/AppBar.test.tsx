import React from "react";
import renderer from "react-test-renderer";
import AppBar from "./AppBar";

it("AppBar renders correctly", () => {
  const tree = renderer.create(<AppBar />).toJSON();
  expect(tree).toMatchSnapshot();
});
