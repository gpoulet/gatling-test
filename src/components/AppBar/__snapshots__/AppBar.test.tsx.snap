// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppBar renders correctly 1`] = `
<nav
  className="navbar navbar-expand navbar-dark bg-dark"
>
  <a
    className="navbar-brand"
    href="/"
  >
    Gatling test
  </a>
  <div
    className="mr-auto navbar-nav"
    onKeyDown={[Function]}
  >
    <a
      className="nav-link"
      data-rb-event-key="/"
      href="/"
      onClick={[Function]}
      onKeyDown={[Function]}
    >
      Home
    </a>
  </div>
</nav>
`;
